cli.js

```diff
@@ -65,6 +65,8 @@ process.on('uncaughtException', function(err) {
       + '\n'
       + '  install react --lock             Stable install, locking existing dependencies\n'
       + '\n'
+      + '  install react --edge             Untable install\n'
```

Comment:
To save space -  `  install react=npm:react --edge   Install a package from a registry to latest unstable\n'